packages/components/organisms/f-checkout/src/services/mapper.js

```diff
@@ -39,8 +39,8 @@ const mapUpdateCheckoutRequest = ({
                         lines: [
                             address.line1 || '',
                             address.line2 || '',
-                            '',
-                            ''
+                            null,
```

Comment:
Should this be checking lines 3 & 4 then setting it to null if it's not populated instead?

Reply:
I believe we always sent an empty string and now always send null. Because we don't actually capture address lines 3 and 4. City and postcode are sent separately in `locality` and `postcode`.